src/statuscake.js

```diff
@@ -2,14 +2,13 @@ const axios = require('axios');
 const debug = require('debug')('statuscake');
 const logError = require('debug')('error');
 const config = require('config');
-const Storage = require('./statuscakedata');
 
 const { username, apikey, baseUrl } = config.statuscake;
 const STATUS_CAKE_BASE_URL = `${baseUrl}?API=${apikey}&Username=${username}`;
 
-const fetchStatuCakeData = () => axios.get(STATUS_CAKE_BASE_URL)
+const fetchStatuCakeData = Storage => axios.get(STATUS_CAKE_BASE_URL)
```

Comment:
you could right `storage` here to differentiate the class from the param IMO

Reply:
Done, thanks 😋